property_table.py

```diff
@@ -1370,6 +1370,26 @@ class PropertyTable:
             "FMantissa9",
             False,
         ),
+        "max_pos": (
+            "MaxParticlePos",
+            3,
+            np.float32,
+            "Mpc",
+            "Maximum position of particles in the halo along each axis.",
```

Comment:
Maybe "Upper bound on the position of particles in the halo along each axis" since it includes the extra 1.001 factor (I assume that's for rounding error?)

Reply:
It was for a rounding error, but actually I'll remove that.